app/views/instrument_relays/_form.html.haml

```diff
@@ -0,0 +1,41 @@
+- content_for :head_content do
+  = javascript_include_tag "power_relay"
+
+= simple_form_for([current_facility, @product]) do |f|
+  = f.input :control_mechanism,
+    collection: options_for_control_mechanism,
+    include_blank: false,
+    selected: @product.control_mechanism || :manual
+
+  .well#power-relay
+    %h3= text("instruments.instrument_fields.relay.label.relay")
+    %p= text("instruments.instrument_fields.relay.intro")
+    = f.simple_fields_for :relay, @product.relay || @product.build_relay do |r|
+      = r.input :type, collection: options_for_relay, include_blank: false
+      = r.input :ip
+      = r.input :ip_port,
+      hint: text("instruments.instrument_fields.relay.instruct.ip_port")
```

Comment:
I find this easier to read with the additional args indented to align with `input`, like below on line 26-29